content/hi/_index.html

```diff
@@ -0,0 +1,61 @@
+---
+title: "प्रोडक्शन-ग्रेड कंटेनर ऑर्केस्ट्रेशन"
+abstract: "स्वचालित कंटेनर डिप्लॉयमेंट, स्केलिंग और प्रबंधन"
+cid: home
+sitemap:
+  priority: 1.0
+---
+
+{{< blocks/section id="oceanNodes" >}}
+{{% blocks/feature image="flower" %}}
+[कुबेरनेट्स]({{< relref "/docs/concepts/overview/what-is-kubernetes" >}}), K8s के रूप में भी जाना जाता है, कंटेनरीकृत एप्लीकेशन के डिप्लॉयमेंट, स्केलिंग और प्रबंधन को स्वचालित करने के लिए एक ओपन-सोर्स सिस्टम है।
+
+यह आसान प्रबंधन और खोज के लिए तार्किक इकाइयों में एक एप्लीकेशन बनाने वाले कंटेनरों को समूहित करता है। कुबेरनेट्स [Google पर उत्पादन कार्यभार चलाने के १५ वर्षों के अनुभव](http://queue.acm.org/detail.cfm?id=2898444), पर निर्माण करता है, जो समुदाय के सर्वोत्तम-नस्लीय विचारों और प्रथाओं के साथ संयुक्त है।
```

Comment:
```suggestion
यह आसान प्रबंधन और खोज के लिए लॉजिकल इकाइयों में एक एप्लीकेशन बनाने वाले कंटेनरों को समूहित करता है। कुबेरनेट्स [Google में उत्पादन कार्यभार चलाने के 15 वर्षों के अनुभव](http://queue.acm.org/detail.cfm?id=2898444), पर निर्माणित है, जो समुदाय के सर्वोत्तम-नस्लीय विचारों और प्रथाओं के साथ संयुक्त है।
```

Reply:
Although we have Devnagri way of writing numbers, Almost all of native speakers too use the Latin way in mordern times. Hence, I think we should go with `15`

Reply:
oh, i guess this was missed by me!! Thanks for pointing out